### PR TITLE
Add mi binding for dc-purview-shir

### PR DESCRIPTION
**Pull request by HarrisonBoyleThomas**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DC-78

### Change description ###
Add identity binding for dc-purview-shir

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by mwallaceHMCTS — APPROVED** [review]

File: apps/dc/dev/base/kustomization.yaml [hunk 1]
```diff
@@ -7,3 +7,4 @@ patches:
   #overlays for dc-purview-shir
   - path: ../../dc-purview-shir/dev.yaml
   - path: ../../serviceaccount/dev.yaml
+  - path: ../../identity/dc-purview-shir-identity-dev.yaml
```
Comment (thomast1906) [inline apps/dc/dev/base/kustomization.yaml:10]: we usually just have the reference as <env>.yaml - this example `dev.yaml`

File: apps/dc/identity/dc-purview-shir-identity.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: aadpodidentity.k8s.io/v1
+kind: AzureIdentity
+metadata:
+  name: dc-purview-shir
```
Comment (thomast1906) [inline apps/dc/identity/dc-purview-shir-identity.yaml:4]: Combine idenity & binding together - as example https://github.com/hmcts/sds-flux-config/blob/master/apps/toffee/identity/apple-azure-identity.yaml

**Review by thomast1906 — CHANGES_REQUESTED** [review]

File: apps/dc/dev/base/kustomization.yaml (hunk 1 above)
Comment (HarrisonBoyleThomas) [reply apps/dc/dev/base/kustomization.yaml]: Updated

File: apps/dc/identity/dc-purview-shir-identity.yaml (hunk 2 above)
Comment (HarrisonBoyleThomas) [reply apps/dc/identity/dc-purview-shir-identity.yaml:4]: Updated

**Review by thomast1906 — APPROVED** [review]
cheers

**Pull request merged**